plugins/convert.py

```diff
@@ -31,17 +31,26 @@ def convert(inp, bot=None):
     parts = input.split()
 
     try:
-        ffrom = parts[0]
-        to = parts[1]
-    except IndexError:
+        parts.remove('TO')
+    except ValueError:
+        pass
+
+    amount = None
+    for x in parts:
+        try:
+            amount = float(x)
+            parts.remove(x)
```

Comment:
try: except: else: pattern please. 

for: else: pattern please.

for x in list(parts):
  try:
    amount = float(x)
  except ValueError:
    pass
  else:
    parts.remove(x)
    break
else:
  amount = 1.0